src/FileConventions/Library.fs

```diff
@@ -7,6 +7,11 @@ open System.Text.RegularExpressions
 
 open Mono
 open Mono.Unix.Native
+open Fsdk
+
+let ymlAssertionError = "Bug: file should be a .yml file"
+let projAssertionError = "Bug: file should be a proj file"
+let fsxAssertionError = "Bug: file was not a F#/C# source file"
```

Comment:
@Mersho if it's an error about F# or C# files, then why do you call it "Fsx Assertion Error"?